src/Resources/views/CRUD/dashboard__action_create.html.twig

```diff
@@ -1,11 +1,11 @@
 {% if admin.subClasses is empty %}
     <a class="btn btn-link btn-flat" href="{{ action.url }}">
-        <i class="fa fa-{{ action.icon }}" aria-hidden="true"></i>
+        <i class="{{ action.icon|parse_icon(true)|raw }}" aria-hidden="true"></i>
```

Comment:
I think you can add `aria-hidden="true"` in the twigExtension (this is a good practice to always add this).
Then you can use `{{ action.icon|parse_icon }}` without the `true` parameter

Same for some others cases.

Reply:
When I add `aria-hidden=true` this how would buttons without a text be displayed? For instance now on the list/mosaic  buttons?

Reply:
If I understand correctly the aria-hidden attribute, it doesn't have any impact on the display.
It's just an accessibility attribute, when you're using a screen reader, it won't read the icon.
But I'm not an expert...